src/validation.cpp

```diff
@@ -3386,7 +3386,14 @@ int GetWitnessCommitmentIndex(const CBlock& block)
     int commitpos = -1;
     if (!block.vtx.empty()) {
         for (size_t o = 0; o < block.vtx[0]->vout.size(); o++) {
-            if (block.vtx[0]->vout[o].scriptPubKey.size() >= 38 && block.vtx[0]->vout[o].scriptPubKey[0] == OP_RETURN && block.vtx[0]->vout[o].scriptPubKey[1] == 0x24 && block.vtx[0]->vout[o].scriptPubKey[2] == 0xaa && block.vtx[0]->vout[o].scriptPubKey[3] == 0x21 && block.vtx[0]->vout[o].scriptPubKey[4] == 0xa9 && block.vtx[0]->vout[o].scriptPubKey[5] == 0xed) {
+            CTxOut vout = block.vtx[0]->vout[o];
+            if (vout.scriptPubKey.size() >= MINIMUM_WITNESS_COMMITMENT
+            && vout.scriptPubKey[0] == OP_RETURN
+            && vout.scriptPubKey[1] == 0x24
+            && vout.scriptPubKey[2] == 0xaa
+            && vout.scriptPubKey[3] == 0x21
+            && vout.scriptPubKey[4] == 0xa9
+            && vout.scriptPubKey[5] == 0xed) {
```

Comment:
style-nit: If you reformat this, might as well make it conform to clang-format:

```diff
diff --git a/src/validation.cpp b/src/validation.cpp
index 28538cfbbd..5411e45d7d 100644
--- a/src/validation.cpp
+++ b/src/validation.cpp
@@ -3387,13 +3387,13 @@ int GetWitnessCommitmentIndex(const CBlock& block)
     if (!block.vtx.empty()) {
         for (size_t o = 0; o < block.vtx[0]->vout.size(); o++) {
             CTxOut vout = block.vtx[0]->vout[o];
-            if (vout.scriptPubKey.size() >= MINIMUM_WITNESS_COMMITMENT
-            && vout.scriptPubKey[0] == OP_RETURN
-            && vout.scriptPubKey[1] == 0x24
-            && vout.scriptPubKey[2] == 0xaa
-            && vout.scriptPubKey[3] == 0x21
-            && vout.scriptPubKey[4] == 0xa9
-            && vout.scriptPubKey[5] == 0xed) {
+            if (vout.scriptPubKey.size() >= MINIMUM_WITNESS_COMMITMENT &&
+                vout.scriptPubKey[0] == OP_RETURN &&
+                vout.scriptPubKey[1] == 0x24 &&
+                vout.scriptPubKey[2] == 0xaa &&
+                vout.scriptPubKey[3] == 0x21 &&
+                vout.scriptPubKey[4] == 0xa9 &&
+                vout.scriptPubKey[5] == 0xed) {
                 commitpos = o;
             }
         }

Reply:
Done.